dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -102,32 +131,113 @@ public CompletableResultCode export(@Nonnull Collection<MetricData> metrics) {
+          break;
+        case LONG_SUM:
+          isDelta =
+              metric.getLongSumData().getAggregationTemporality() == AggregationTemporality.DELTA;
+          metricLines.addAll(serializer.createLongSumLines(metric, isDelta));
+          break;
+        case DOUBLE_GAUGE:
+          metricLines.addAll(serializer.createDoubleGaugeLines(metric));
+          break;
+        case DOUBLE_SUM:
+          isDelta =
+              metric.getDoubleSumData().getAggregationTemporality() == AggregationTemporality.DELTA;
+          metricLines.addAll(serializer.createDoubleSumLines(metric, isDelta));
+          break;
+        case SUMMARY:
+          metricLines.addAll(serializer.createDoubleSummaryLines(metric));
+          break;
+        case HISTOGRAM:
+          metricLines.addAll(serializer.createDoubleHistogramLines(metric));
+          break;
+        default:
+          logger.warning(
+              String.format(
+                  "Tried to serialize metric of type %s. The Dynatrace metrics exporter does not handle metrics of that type at the time.",
+                  metric.getType().toString()));
+          break;
+      }
+    }
+
+    return metricLines;
+  }
+
   @VisibleForTesting
   protected CompletableResultCode export(
       Collection<MetricData> metrics, HttpURLConnection connection) {
-    String mintMetricsMessage = MetricAdapter.toMint(metrics).serialize();
-    logger.log(Level.FINEST, "Exporting: {0}", mintMetricsMessage);
-    try {
-      connection.setRequestMethod("POST");
-      connection.setRequestProperty("Accept", "*/*; q=0");
-      connection.setRequestProperty("Authorization", "Api-Token " + apiToken);
-      connection.setRequestProperty("Content-Type", "text/plain; charset=utf-8");
-      connection.setDoOutput(true);
-      try (final OutputStream outputStream = connection.getOutputStream()) {
-        outputStream.write(mintMetricsMessage.getBytes(StandardCharsets.UTF_8));
+
+    List<String> metricLines = makeMetricLines(metrics);
+    for (List<String> partition :
+        Lists.partition(metricLines, DynatraceMetricApiConstants.getPayloadLinesLimit())) {
+      CompletableResultCode resultCode;
+      String joinedMetricLines = Joiner.on('\n').join(partition);
+
+      if (logger.isLoggable(Level.FINER)) {
+        logger.finer(String.format("Exporting metrics:\n%s", joinedMetricLines));
       }
-      int code = connection.getResponseCode();
-      if (code != 202) {
-        logger.log(Level.WARNING, "Received error code {0} from server", code);
-        return CompletableResultCode.ofFailure();
+      try {
+        connection.setRequestMethod("POST");
+        connection.setRequestProperty("Accept", "*/*; q=0");
+        if (this.apiToken != null) {
+          connection.setRequestProperty("Authorization", "Api-Token " + apiToken);
+        }
+        connection.setRequestProperty("Content-Type", "text/plain; charset=utf-8");
```

Comment:
```suggestion
        connection.setRequestProperty("Content-Type", "text/plain; charset=utf-8");
        connection.setRequestProperty("User-Agent", "opentelemetry-metric-java");
```
Let's also add this one here.